app/controllers/application.js

```diff
@@ -8,13 +8,21 @@ export default Ember.Controller.extend({
   config,
 
   app_id: config.APP.ANDROID_APP_ID,
+  ios_app_id: config.APP.IOS_APP_ID,
 
   initSubscriptions: Ember.on('init', function() {
     if (this.session.get("isLoggedIn")) {
       this.send('setSubscriptions');
     }
   }),
 
+  appIcon: Ember.computed('config', function() {
+    if (config.APP.NAME === 'app.goodcity') {
```

Comment:
Would it be better to have config.APP.TITLE defined?

Reply:
@steveyken , you mean taking `config.APP.NAME` in a separate variable like `let appTitle = config.APP.NAME` and using it in if condition or declaring different field `config.APP.TITLE` in environment file ?

Reply:
The latter one - define `config.APP.TITLE` in environment file

Reply:
Added 👍